vowpalwabbit/active.cc

```diff
@@ -112,7 +114,7 @@ void output_and_account_example(vw& all, active& a, example& ec)
   all.sd->update(ec.test_only, ld.label != FLT_MAX, ec.loss, ec.weight, ec.get_num_features());
   if (ld.label != FLT_MAX && !ec.test_only)
   { all.sd->weighted_labels += (static_cast<double>(ld.label)) * static_cast<double>(ec.weight); }
-  all.sd->weighted_unlabeled_examples += ld.label == FLT_MAX ? static_cast<double>(ec.weight) : 0.0;
+  // all.sd->weighted_unlabeled_examples += ld.label == FLT_MAX ? static_cast<double>(ec.weight) : 0.0;
```

Comment:
whats the deal with this comment?

Reply:
Oh great catch thank you! This line is redundant because of line 114 and i was going to open a PR to remove it. However, I accidentally added it here